Content.Client/Commands/AtmosDebugCommands.cs

```diff
@@ -0,0 +1,130 @@
+using Content.Client.GameObjects.EntitySystems;
+using Robust.Shared.GameObjects.Systems;
+using Content.Shared.Atmos;
+using System;
+
+namespace Content.Client.Commands
+{
+    [UsedImplicitly]
+    internal sealed class AtvRangeCommand : IConsoleCommand
+    {
+        public string Command => "atvrange";
+        public string Description => "Sets the atmos debug range (as two floats, start [red] and end [blue])";
+        public string Help => "atvrange <start> <end>";
+        public bool Execute(IDebugConsole console, params string[] args)
+        {
+            if (args.Length != 2)
+            {
+                console.AddLine(Help);
+                return false;
+            }
+            if (!float.TryParse(args[0], out var xStart))
+            {
+                console.AddLine("Bad float START");
+                return false;
+            }
+            if (!float.TryParse(args[1], out var xEnd))
+            {
+                console.AddLine("Bad float END");
+                return false;
+            }
+            if (xStart == xEnd)
+            {
+                console.AddLine("Scale cannot be zero, as this would cause a division by zero in AtmosDebugOverlay.");
+                return false;
+            }
+            var sys = EntitySystem.Get<AtmosDebugOverlaySystem>();
+            sys.CfgBase = xStart;
+            sys.CfgScale = xEnd - xStart;
+            return false;
+        }
+    }
+
+    [UsedImplicitly]
+    internal sealed class AtvModeCommand : IConsoleCommand
+    {
+        public string Command => "atvmode";
+        public string Description => "Sets the atmos debug mode. This will automatically reset the scale.";
+        public string Help => "atvmode <TotalMoles/GasMoles/Temperature> [<gas ID (for GasMoles)>]";
+        public bool Execute(IDebugConsole console, params string[] args)
+        {
+            if (args.Length < 1)
+            {
+                console.AddLine(Help);
+                return false;
+            }
+            if (!Enum.TryParse<AtmosDebugOverlayMode>(args[0], out var xMode))
+            {
+                console.AddLine("Invalid mode");
+                return false;
+            }
+            int xSpecificGas = 0;
+            float xBase = 0;
+            float xScale = Atmospherics.MolesCellStandard * 2;
+            if (xMode == AtmosDebugOverlayMode.GasMoles)
+            {
+                if (args.Length != 2)
+                {
+                    console.AddLine("A target gas must be provided for this mode.");
+                    return false;
+                }
+                if (!int.TryParse(args[1], out xSpecificGas))
+                {
+                    console.AddLine("Gas ID not parsable.");
+                    return false;
+                }
+                if ((xSpecificGas < 0) || (xSpecificGas >= Atmospherics.TotalNumberOfGases))
+                {
+                    console.AddLine("OUT OF RANGE");
```

Comment:
Might want to make this a tiny bit more descriptive

Reply:
Darn, the backfire of memes. Really though this should probably also accept gas names and stuff, will do